lib/slack_poster.rb

```diff
@@ -30,7 +29,7 @@ def send_request(message)
 
 private
 
-  attr_reader :postable_day, :today
+  attr_reader :today
```

Comment:
Do you mean testing the `if !Date.today.saturday? && !Date.today.sunday?` line? Ins't that just testing that Ruby methods work? The way the Seal works now, the Heroku scheduler runs every day and it will simply stop at that line when it's a weekend.